### ISSUE TITLE
Value Detector seems overly powerful.

### ISSUE THREAD

**Issue by Hecter94** [endless-sky/endless-sky#8312]
### Problem Description

Compared to every other scanner outfit, the value detector gives superior stats with little to no justification as to why. 
![image](https://user-images.githubusercontent.com/15916854/217053046-91fab50a-5df0-43e2-9082-c77ef28d802c.png)

It's tactical scan range/mass is 319, compared to the Outskirts Gauger at 162.5 or the Salvage Scanner at 130. 
It's Scan/Mass is 30.67 compared to the Outskirts Gauger at 21.13 or the Salvage Scanner at 12

<details>
<summary>Chart Showing Total Scan Power over Mass</summary>

![image](https://user-images.githubusercontent.com/15916854/217025906-c6a4fb14-299c-4078-9141-2f458530db2d.png)

</details>

<details>
<summary>Chart Showing Tactical Scan Range over Mass</summary>

![image](https://user-images.githubusercontent.com/15916854/217028992-4296bfc2-7d97-466c-9454-8b4819acd93f.png)

</details>

It's a straight upgrade to the scanning module, having 67 more tactical scan power and 63 more cargo scan power for the same outfit usage.

Given that it is not license locked, and is easily obtainable just by purchasing it, and is described as "Rather Rudimentary" I would expect it to have values more in-line with the lower end of Hai technology, perhaps a bit better than human scanners, not easily besting Coalition/Remnant scanners. 

### Desired Solution

I would suggest dropping it's stats as follows:

cargo scan power: 88->30
tactical scan power: 92->15

This makes it effectively a sidegrade to the Coalition Scanning Module, with better cargo scan at the expense of worse tactical scan and no outfit scan for the same outfit space. 
Given that the Scanning Module requires the Heliarch license, this may still be too powerful, but it maintains its position as being a very good scanner, especially considering how early it can be accessed.

### Alternative Approaches

Alternatively, it could keep its current stats but be given some other downside. 
Perhaps increasing its mass to ~12, or giving it energy consumption or heat generation as befitting it's rudimentary nature. 

It would still be an extremely powerful scanner, but would come with downsides that more advanced tech doesn't. 
Even if we did go this route, it's actual scan power should still probably be lowered.

**Comment by Zitchas** [comment]
The Heliarchs maintain an excessive control over any dual-use (aka could be used for military purposes) technology, so I don't think the "ease of access" of technology under their control is particularly useful for comparison.

I don't see a problem with them having a good cargo scanner, possibly even the best cargo scanner (I'd imagine that the Korath Exiles should probably have a pretty good one, too).

That being said, the Value Detector was originally added in https://github.com/endless-sky/endless-sky/pull/7019 , where it had the following statistics:
```
outfit "Value Detector"
	category "Systems"
	cost 260000
	thumbnail "outfit/value detector"
	"mass" 3
	"outfit space" -3
	"cargo scan power" 22
	"cargo scan speed" 4
	"tactical scan power" 27
	description "The Unfettered developed this scanner in order to determine which ships are worth plundering, or which lack the capacity to stand up to their ion weapons. Some Hai consider it rather rudimentary, but it is sufficient to identify the cargo on a ship from afar and provide more detailed information about the tactical condition of enemy vessels. This dual function is useful when fielding experimental weaponry."
```

It received a fairly massive buff in https://github.com/endless-sky/endless-sky/pull/7649 . 

I note, looking through that PR, that the Value Detector was the only place something received a buff to scan *power*, everything else merely gets a buff to scan *speed*. It's here that it gets bumped up to the values listed in this issue. There does not, however, appear to be much - if any - discussion in either PR about the values of this outfit.

**Comment by Hecter94** [comment]
> edit: As an aside, the chart at the top of this issue doesn't appear to have the basic cargo scanner. That might be useful as a reference point.

Whoops, thanks for pointing that out. Edited the chart to include any scanner with tactical OR cargo scan, rather than just ones with tactical scan.

**Comment by Zitchas** [comment]
I think I would opt for values along the lines of:

Cargo Scan 40 (maybe even 50)
Tactical Scan 30

And I'd tweak the description to remove the reference to it being rudimentary.

That being said, if the people working on the Hai project have plans to unveil a higher power Hai scanner later in the campaign, then I'd suggest:

Cargo Scan 35
Tactical Scan 24

And leaving the description as-is.

Given the general opinion of the uselessness of scanners and tactical scanners in particular, I'd prefer to have this as less of a downgrade compared to the tactical scanner + outfit scanner combo. Someone swapping to it will still have a decreased range, but at least it has the same scan/space ratio so they can stack it instead of having to go back to the tactical scanner. That combo takes 5 space to give C 9 & T 32, with Value detectors (using my values) the player can either choose to save space and go down to 3 with C35/T24 (so they lose a little tactical range in exchange for a lot more cargo scan), or they can spend an extra unit of space to go up to 6 giving C70 & T48, which is a significant upgrade.

**Comment by Hecter94** [comment]
Tactical Scan of 30 still seems too high if it's going to have the best cargo scan in the game.

That would make it basically on par with the tactical scanner, having an insignificant amount less tactical scan range (18), with 29% less outfit space on top of being the best cargo scanner in the game. (13.33 cargo scan/mass, second best being the cargo scanner at 9 scan/mass)

It would also be the best tactical scanner in the game in terms of range/mass.
![image](https://user-images.githubusercontent.com/15916854/217077563-77c04c11-00e0-444e-8b37-25afc78f749a.png)

If it's going to be such a good tactical scanner, it really shouldn't have cargo scan at all, and vice versa.
Otherwise, both should be lowered down so that it's not the best in either area, but gives both to compensate. 

If you want to keep Cargo Scan at 35/40, Tactical Scan would have to be dropped down to at least 15, giving it 129 range/mass compared to the tactical scanner at 141 range/mass. 

That would make it the best cargo scanner in the game, and fifth best tactical scanner, basically on par with the salvage scanner:
![image](https://user-images.githubusercontent.com/15916854/217079000-d4fcb5e4-0736-4769-a1f6-772a590d5063.png)

In regards to player choice in this case, you can choose to use a tactical scanner and a cargo scanner to get 566 tactical scan range, 9 cargo scan power and 5 cargo scan efficiency for 5 outfit space.
Or a Value detector to get 387 tactical scan range, 40 cargo scan power and 16 cargo scan efficiency, for 3 outfit space.

So it's still a pretty significant improvement over human stuff, much better cargo scan in less space at the cost of 179 tactical scan range, it just isn't a "straight" upgrade.

It would also be much more comparable to the Scanning Module in this case. 
Both of them cost 3 outfit space, scanning module gives 37 more tactical scan range and 25 outfit scan, at the expense of 15 cargo scan power.
So you're effectively trading 25 outfit scan for 15 cargo scan. 

I definitely don't think it should be the best cargo scanner *and* the best tactical scanner in the game, at least not without some sacrifices elsewhere to justify that.

**Comment by Arachi-Lover** [comment]
>The Heliarchs maintain an excessive control over any dual-use (aka could be used for military purposes) technology, so I don't think the "ease of access" of technology under their control is particularly useful for comparison.

Don't get what this is saying; it is useful and relevant as the Heliarch Scanning Module is them building a general purpose scanner that's standard issue to Heliarch ships, much like how the Unfettered's Value Detector is them building a general purpose scanner that's supposed to be commonly employed.
Same with how one can compare Heliarch ion weapons with other ion weapons like the Ion Cannon despite the Heliarch control over their weapons tech and other weapons being available for purchase right off the bat.

**Comment by Zitchas** [comment]
To be honest, I don't find tactical scan below range 500 to be useful. In such close quarters I've got better things to do than analyze how my enemy's energy supply is doing. As such, 

Regarding the Unfettered's development of scanning resources, I don't see this so much as the Unfettered developing it, as the Unfettered being the sub-branch of the Hai who took already available technology their society has, and put it together for use. In other words, this isn't *unfettered* technology, this is *Hai* technology, just that the Unfettered are the only part of the Hai that see a point in manufacturing it. This is the sort of technology that the Unfettered might be primarily using at the start of the game, but post-Hai Reveal when crime is much more to the fore-front of Hai thinking, they start manufacturing too (or put out a better, more professional, replacement for it).

The Hai are, in some respects, in the same situation as the Wanderers wherein they aren't particularly militarized currently, but they have had periods of their past where they have been. The Unfettered see a use for bringing old tech back into use, the Hai don't, at least up until Hai Reveal. They clearly have a very stable government with records and as best we can tell, functioning government going back to their Empire days, and thus have just as much justification for pulling old formerly useful tech out of retirement as the Wanderers do. 

----------

I find these tables to be somewhat deceptive. They are, as best I can tell, completely accurate if comparing the performance of a single outfit by itself. But all those "range per mass" calculations are, unless I'm mistaken, are largely wrong if looking at two copies of said outfit, and utterly wrong when looking at three or four of them. I, personally, generally aim for tactical scan at around 1000-1500 range, because I want to look at a ship's info and make tactical decisions before my target is actually shooting at me with primary weapons.

As such, I think we need to have a larger discussion about expectations.

1. Are tactical scanners something that are expected to be used primarily in single-outfit situations? Or in multiple outfit situations?
2. Is "ensuring scanning outfits lose efficiency as they are stacked" still something that we want and is actually beneficial to the game?
3. What is the purpose of preventing stacking? 
4. Why are scanners and ramscoops "special" in that they are exclusively the only things that are so crucial to penalize stacking that we build this penalty into the very mechanics of the code? There's a lot of things that arguably should decrease in effectiveness as they are stacked, yet these two things which impact two areas of the game that are downplayed, have these restrictions. (fuel is devalued to the point that random passing strangers will dependably go out of their way to give us some of theirs, and all ships universally regenerate it for *free*. So why make it extra hard to reduce the delay further?) Scanning, likewise, is occasionally a mission required thing, but is largely considered to be of zero combat value and a waste of space. So why put effort into making it hard to do better?

The fact that the comparison tables are explicitly sidelining the scan power and focusing on range per space highlights the fact that for some, perhaps many of us, that range/ton is the actual important value, and that once the desired range for that outfit is determined, the power is just derived from it. That implies that we don't actually care about the power. However, rang/space doesn't work because the mechanics are specifically built to ensure that every successive unit gives less range. I would be curious what the table looked like if it was measuring the increased range/space provided by the outfit if the ship in question *already has*, say, 50 scan power.

**Comment by Arachi-Lover** [comment]
If it was old tech it'd be more reasonable (though I imagine it'd still need brought in line), but with the current description it's made clear it's an Unfettered development, and the Unfettered are very a recent group in Hai history, so need to update the description to go for that angle.

**Comment by Hecter94** [comment]
Since we seem to be in agreement that the value detector is too powerful, I've gone ahead and opened a PR where we can discuss specifics:
https://github.com/endless-sky/endless-sky/pull/8318

**Comment by EjoThims** [comment]
It very clearly needs a nerf from its present state, but I'm a bit confused why there's no tables/ break down of it's stats previous to the PR in which it was adjusted. As far as I can tell (other than the one the PR doesn't touch), that's roughly equivalent to suggested here, but with a more even split instead.

**Comment by Zitchas** [comment]
@Hurleveur Just to ping you into the issue. :)

A general thought: Until such time as we decide that range/ton is the metric to use (and we get rid of the square root mechanic so that range/space is actually a consistent value, let's stick to power/space, as that *is* a consistent value.

**Comment by UnorderedSigh** [comment]
I think this was done because the Unfettered campaign needed scanners that actually worked, and scanners in Endless Sky have been nerfed into oblivion. If we merge #8288, then the Value Detector won't need insane stats for the Unfettered campaign to work.

**Comment by UnorderedSigh** [comment]
To those unaware, #8288 adjusts the formulas used for scanning, so scanners work about as well as they used to, on ships around the size of a medium warship. Having a big ship is still a "get out of jail free" card, but scanning one is within the player's ability, even though AI bugs keep the Navy from doing scanning them.

It would be a *very, very* good idea to merge #8288 before the 0.10.0 release, so that scanning works again.

**Comment by Zitchas** [comment]
In light of that, I would suggest that we get the mechanics of scanners nailed down first. Only then can they be balanced appropriately. Until then, I think leaving them as-is in order to provide needed functionality takes precedence over lore or balance concerns.

**Comment by Hurleveur** [comment]
I will adjust the Unfettered campaign to give you 2 value detectors and 4 cargo scanners, so this change is not problematic.

It mostly brings it closer into the niche that was planned for it, so all is good. We should probably adjust it a bit once the formulas change, but if it ends up scanning faster than now with those changes, all the better. The tactical scanning was, in all honesty, too high, the heat gives it more unfettered-ness and the lower efficiency is fine, the point is to do it from afar. Yes, it is a bit slow, but so are all other scanners, so that will be fixed in another way.

**Comment by UnorderedSigh** [comment]
What is missing from this discussion is that the balance and lore issue is actually the Hai's ability to scan, which is based on their ship designs. The Value Detector, in my opinion, should be balanced based on stats and the Hai tier, which is independent of whether scanners work in the game at all. Likewise, the tuning in #8288 has to be based on the only situations where scanning actually matters: early game ships before the player has godlike abilities to avoid detection, and scanning missions.

The lore and balance issue for the Hai is actually how well their surveillance ship designs can scan. If we have Value Detectors that are consistent with the other scanners, and scanning formulae that work, we can tweak the Hai designs accordingly.

Put simply, I'd rather this PR and #8288 both be merged as soon as possible, in whatever order gets them in quickest. Then someone should do a follow-up PR to tweak Hai surveillance designs to be lore-appropriate and balanced.

**Comment by Hurleveur** [comment]
When I come back I'll add a way to have tactical scanner that provides only energy of ships (and maybe their max range as well) and boost the scanning accordingly, but in the meantime this is fine.
We can also boost the cargo scanning at that point, going with the logic this is Hai tech that was somewhat overclocked

**Comment by Hecter94** [comment]
Closing this as #8318 was merged.
